app/presenters/summary/sections/other_income_details.rb

```diff
@@ -23,7 +23,7 @@ def answers
 
       private
 
-      def manage_without_income_subject
+      def subject_type
```

Comment:
Overwrite the base class's subject_type to applicant AND partner when partner included. For this summary card only.